### PR TITLE
Update view-reports-for-mdo.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information about URL click protection action

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4501

File: microsoft-365/security/office-365-security/view-reports-for-mdo.md [hunk 1]
```diff
@@ -278,8 +278,8 @@ The **URL threat protection** report has two aggregated views that are refreshed
 - **URL click protection action**: Shows the number of URL clicks by users in the organization and the results of the click:
 
   - **Blocked** (the user was blocked from navigating to the URL)
-  - **Blocked and clicked through**
-  - **Clicked through during scan**
+  - **Blocked and clicked through** (the user choose to continue navigating to the URL)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-reports-for-mdo.md]:
```suggestion
  - **Blocked and clicked through** (the user has chosen to continue navigating to the URL)
```

File: microsoft-365/security/office-365-security/view-reports-for-mdo.md [hunk 2]
```diff
@@ -278,8 +278,8 @@ The **URL threat protection** report has two aggregated views that are refreshed
...
+  - **Clicked through during scan** (the user click on the link before the scan is complete)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-reports-for-mdo.md]:
```suggestion
  - **Clicked through during scan** (the user has clicked on the link before the scan was complete)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Pull request merged**